compat_policy.go

```diff
@@ -0,0 +1,89 @@
+package nftables
+
+import (
+	"fmt"
+
+	"github.com/google/nftables/expr"
+	"golang.org/x/sys/unix"
+)
+
+const NFT_RULE_COMPAT_F_INV uint32 = (1 << 1)
```

Comment:
Can you make these two constants unexported please? (E.g. by lower-casing the “nft”)

We should avoid adding anything to the public API surface of the nftables package unless users really need it :)

Reply:
done